tutorials/navigation/navigation_introduction_2d.rst

```diff
@@ -100,7 +100,7 @@ It uses the NavigationServer2D and a NavigationAgent2D for path movement.
    .. image:: img/nav_2d_min_setup_step1.png
 
 #. Define the movable navigation area with the NavigationPolygon draw tool. Then click
-   the `Bake NavigationPolygon`` button on the toolbar.
+   the ``Bake NavigationPolygon`` button on the toolbar.
```

Comment:
```suggestion
   the :button:`Bake NavigationPolygon` button on the toolbar.
```
New syntax while we're here

Reply:
Created #11257 to change other occurrences.